lib/elixir/lib/kernel.ex

```diff
@@ -2746,17 +2810,21 @@ defmodule Kernel do
   end
 
   defp build_if(condition, do: do_clause, else: else_clause) do
-    optimize_boolean(quote do
-      case unquote(condition) do
-        x when x in [false, nil] -> unquote(else_clause)
-        _ -> unquote(do_clause)
+    optimize_boolean(
+      quote do
+        case unquote(condition) do
+          x when x in [false, nil] -> unquote(else_clause)
+          _ -> unquote(do_clause)
+        end
       end
-    end)
+    )
   end
 
   defp build_if(_condition, _arguments) do
-    raise(ArgumentError, "invalid or duplicate keys for if, only \"do\" " <>
-      "and an optional \"else\" are permitted")
+    raise(
+      ArgumentError,
+      "invalid or duplicate keys for if, only \"do\" " <> "and an optional \"else\" are permitted"
```

Comment:
Please assign the message to a variable in the previous line and remove concatenation.

Reply:
Also remove the parens around raise.